lib/completion.ts

```diff
@@ -193,7 +193,8 @@ export class Completion implements CompletionInstance {
       return (this.customCompletionFunction as FallbackCompletionFunction)(
         current,
         argv,
-        () => this.defaultCompletion(args, argv, current, done),
+        (onCompleted = done) =>
```

Comment:
We should add a test:

`it('allows custom completion to be combined with default completion, using filter')`.